### ISSUE TITLE
Adding a reservation that spans two months is difficult

### ISSUE THREAD

**Issue by jevers76** [dolson334/banterCustomer#9]
A possible improvement might be to add buttons to skip forward or backwards one week make the entire reservation on a single page

**Comment by dolson334** [comment]
@jevers76 currently, you can select a checkIn then click "Next Month" and still can select the check out. Have you tried that?

**Comment by jevers76** [comment]
Yes, selecting Next Month does work.  This is only a suggested improvement.  It would also help in viewing existing reservations allowing to see the entire reservation at once.
